icons/disney.svg

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
```

Comment:
Could you move this icon to the other folder so my scripts will pick it up?

Reply:
ah, that explains the issue i had with the last icons i added #282